hbase-server/src/main/java/org/apache/hadoop/hbase/replication/HReplicationServer.java

```diff
@@ -50,7 +72,7 @@
  */
 @InterfaceAudience.Private
 @SuppressWarnings({ "deprecation"})
-public class HReplicationServer extends Thread implements Server {
+public class HReplicationServer extends Thread implements Server, ReplicationSourceController  {
```

Comment:
Looks like ReplicationServer already committed (why have the 'H' prefix?)

Reply:
Only for the feature branch HBASE-24666.